src/index.js

```diff
@@ -1,41 +1,31 @@
 import React, { Component } from 'react';
 import { AutoComplete, MenuItem } from 'material-ui';
 import Marker from 'material-ui/svg-icons/maps/place';
+import { google } from 'global';
```

Comment:
Instead of using this line `import { google } from 'global';` (please remove)

Replace `const google = window.google;` with `const { google } = global;`